src/Jobs/Feeds/Feed.php

```diff
@@ -99,13 +93,11 @@ public function handle(ProductRepository $productRepository, S3ClientInterface $
 
     abstract protected function productToFeedRow(Product $product);
 
-    /**
-     * @return \Illuminate\Database\Eloquent\Builder
-     */
-    protected function getProductQuery()
+    protected function getProductQuery(): Builder
     {
         return $this->productRepository->newQuery()
             ->where('is_for_sale', 1)
+            ->where('id', '<', 59)
```

Comment:
Careful now 😉 
Can you remove this line?

Reply:
Oh, forgot about it.